Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -44,8 +43,6 @@ def pr_automerge
       "do not merge",
       "new formula",
       "automerge-skip",
-      "linux-only",
-      "linux to homebrew-core",
```

Comment:
I think we still want to ignore `linux to homebrew-core`. The reason why this was added to the ignore list is that we don't want @BrewTestBot to automatically merge PRs that make Linux-only changes to a formula that still has old (e.g. `high_sierra`) bottles. I'm sure we'll still have many PRs like that (see, e.g., Homebrew/homebrew-core#80552), and we'll want to merge those manually/dispatch bottles manually.